package.json

```diff
@@ -1,17 +1,20 @@
 {
   "name": "babel-plugin-react-cssmoduleify",
-  "version": "0.9.5",
+  "version": "1.0.0-beta",
   "description": "Babel plugin to transform traditional classNames to CSS Modules",
   "main": "lib/index.js",
-  "jsnext:main": "index.js",
+  "jsnext:main": "src/index.js",
```

Comment:
This is why I think lib should have multiple builds in it :). That would make it consumable by rollup even with flow types. Just sayin :)

```
lib/
  - browser
  - node 4 https://github.com/jbach/babel-preset-es2015-node4
  - node 5 // https://github.com/alekseykulikov/babel-preset-es2015-node5
```